crates/axfs_vfs/src/lib.rs

```
@@ -254,6 +255,20 @@ pub trait VfsNodeOps: Send + Sync {
 
         Ok(())
     }
+
+    /// Ioctl for device, e.g. pty device
+    fn ioctl(&self, _cmd: usize, _arg: usize) -> VfsResult<usize> {
+        Err(AxError::NoTty)
+    }
+
+    /// Poll for device, e.g. pty device
```

There is a function for `VfsNodeOps`, is the comment right?

---

And I think this function should return a error as `unimplent` for best?